README.asciidoc

```diff
@@ -22,6 +22,7 @@ image:https://hakuneko.download/assets/images/discord-small.png[link=https://dis
 
 image:https://img.shields.io/github/downloads/manga-download/hakuneko/latest/total?label=HakuNeko%20%28Stable%29&logo=github[link=https://github.com/manga-download/hakuneko/releases/latest, title="Download the latest stable release of HakuNeko", window="_blank"]
 image:https://img.shields.io/github/downloads-pre/manga-download/hakuneko/latest/total?color=blue&label=HakuNeko%20%28Nightly%29&logo=azure-devops[link=https://github.com/manga-download/hakuneko/releases, title="Download the latest pre-release (nightly build) of HakuNeko", window="_blank"]
+image:https://img.shields.io/chocolatey/dt/hakunekonightly?color=blue&label=Chocolatey%20package[Chocolatey]
```

Comment:
May set a link to the corresponding repository, e.g.
https://community.chocolatey.org/packages?q=hakuneko

Reply:
Great idea @ronny1982 ! I have added a commit in order to set the link :wink: